doc/branding.md

```diff
@@ -19,6 +19,43 @@ let ConfigDefaults = {
 
 ```
 
+## Dynamic branding configuration
+
+When using configuration from site config, it is recommended to use empty SVG images as default.
```

Comment:
The branding can be configured using a dynamic configuration (aka site config).
This means that the actual logos are loaded from the site configuration after the app is started.
For that reason, it is recommended to use empty SVG logos as pre-configured default.
The empty logos are provided in the ASAB WebUI Demo app.